packages/component-library/src/components/plmg-separator/plmg-separator.stories.js

```diff
@@ -0,0 +1,41 @@
+import * as Utils from '../../stories/StencilStorybookUtils';
+
+export default {
+  title: 'Component/Separator',
+  parameters: {},
+  decorators: [],
+  argTypes: {
+    direction: {
+      control: { type: 'select' },
+      options: ['horizontal', 'vertical'],
+    },
+    thickness: {
+      control: { type: 'select' },
+      options: ['thin', 'thick'],
+    },
+  },
+};
+
+const PROPS = ['direction', 'thickness'];
+const EVENTS = [];
+const CSS_VARS = [];
+
+const Template = (args) => {
+  const el = document.createElement('plmg-separator');
+  // provides a fixed sized wrapper for the component to inherit height and width from
+  const wrapper = document.createElement('div');
+  wrapper.style.height = '200px';
+  wrapper.style.width = '200px';
+  Utils.bindProps(el, PROPS, args);
+  Utils.bindEvents(el, EVENTS, args);
+  Utils.bindStyles(el, CSS_VARS, args);
+  wrapper.appendChild(el);
+  return wrapper;
+};
+
+export const Primary = Template.bind({});
+Primary.storyName = 'Separator';
+Primary.args = {
+  direction: 'horizontal',
+  thickness: 'thin',
+};
```

Comment:
Could you add 2 more stories: 
1. show all variations of `direction` control
2. show all variations of `thickness` control

It will make our life easier when writing the documentation on Zeroheight.
Thank you :pray:

Reply:
+1 to this